src/server/protocol.d.ts

```diff
@@ -1216,4 +1216,38 @@ declare namespace ts.server.protocol {
     export interface NavBarResponse extends Response {
         body?: NavigationBarItem[];
     }
+
+    export interface TsdOptions {

```

Comment:
Actually the definitions are shared in two places. In the future if something else for typing acquisition comes out, they should not be used anyways, as they are specifically for tsd.